src/components/banner/banner.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./banner.scss";
+
+export const Banner = ({ Color, Title, Link }) => {
```

Comment:
It's good practice to use camelCase for the props. PascalCase it's common for the ComponentNames
```suggestion
export const Banner = ({ color, title, link }) => {
```